low-level/files/iec_mapping/ha

```diff
@@ -0,0 +1,4 @@
+0080010001,NodeFailure,The cluster has lost srvnode-1 server; System is running in degraded mode.
```

Comment:
Do you really want 0080010001 & 0080010002 as separate entries. In description you may manage variables like srvnode-1 or srvnode-2.
same comment for 0080020001, 0080020002

Reply:
`srvnode-1`
i thought it will be constant string.
i am not able to find any example for dynamic var

can you share example to use variable

Reply:
Looks like we can change description

I will update it to use same iem for log
```
Oct 21 14:01:19 sm6-r4 python3[220892]: IEC:WS0080010001:The cluster has lost <HOST1> server; System is running in degraded mode.
Oct 21 14:01:49 sm6-r4 python3[220892]: IEC:WS0080010001:Test string only: The cluster has lost <HOST1> server; System is running in degraded mode.
Oct 21 14:02:07 sm6-r4 python3[220892]: IEC:WS0080010001
Oct 21 14:02:15 sm6-r4 python3[220892]: IEC:WS0080010001:
Oct 21 14:04:50 sm6-r4 python3[220892]: IEC:WS0080010001:{'node':'srvnode-1','msg':'down'}
```

Here also tried empty string bit not got default msg.
but we are good to update ha iem

will push in next commit

Reply:
Done